components/SpeakersSection.js

```diff
@@ -108,6 +111,23 @@ function Speaker ({ name, title, photo }) {
           letter-spacing: 0.05em;
           opacity: 0.5;
         }
+
+        .speaker-info {
+          max-width: 270px;
+          margin: auto auto;
```

Comment:
For vertical margin, using `auto` doesn’t take any effect. It is better to set the vertical margin to zero:

```css
margin: 0 auto;
```